client/components/domains/register-domain-step/index.jsx

```diff
@@ -589,6 +590,23 @@ class RegisterDomainStep extends Component {
 		);
 	}
 
+	renderBrowserChrome() {
+		return (
+			<div className="register-domain-step__domain-side-content-container-browser-chrome">
```

Comment:
If you change the method name, I think it would be nice to change these CSS classes as well to something like `__domain-side-content-container-domain-explanation-image`.